docs/changelog.rst

```diff
@@ -14,6 +14,11 @@ Fixed
 
 - Partially generated explicit examples are always valid and can be used in requests. `#582`_
 
+Added
```

Comment:
Could you, please, move this block above under the `Unreleased` section?

Also, one empty line should be added after the `~~~~~` line and there should be a link added to the bottom of the file:

```_#458: https://github.com/kiwicom/schemathesis/issues/458```